source/common/http/conn_manager_impl.cc

```diff
@@ -792,6 +792,10 @@ absl::optional<uint64_t> ConnectionManagerImpl::HttpStreamIdProviderImpl::toInte
       *parent_.request_headers_);
 }
 
+namespace {
+static const std::string kRouteFactoryName = "envoy.route_config_update_requester.default";
```

Comment:
```suggestion
constexpr absl::string_view kRouteFactoryName = "envoy.route_config_update_requester.default";
```

Reply:
Done, but kept `static`. LMK if you think it should not be.

Reply:
static is redundant in this context. It is not going to hurt but it is also not doing anything either. Anonymous namespace makes anything in it translation unit local.

Reply:
Removed.